core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/internal/unmarshall/JsonUnmarshallingParser.java

```diff
@@ -159,7 +159,7 @@ private List<Object> parseList(JsonUnmarshallerContext c, SdkField<?> field, Jso
         if (isScalarType(marshallingType)) {
             MarshallingKnownType marshallingKnownType = marshallingType.getKnownType();
             while (currentToken != JsonToken.END_ARRAY) {
-                result.add(simpleValueFor(field, marshallingKnownType, c, parser, currentToken));
+                result.add(simpleValueFor(memberInfo, marshallingKnownType, c, parser, currentToken));
```

Comment:
- Can we please update the PR in modification about this change.
- Also could you please let me know the test case which covers this change for List parse and Map parse

Reply:
1. [List of timestamps](https://github.com/aws/aws-sdk-java-v2/blob/6747fa2abf24d9a0b686f6be3dbe2710719677eb/test/protocol-tests-core/src/main/resources/software/amazon/awssdk/protocol/suites/cases/json-core-output.json#L531)
2. [Map of timepstamps](https://github.com/aws/aws-sdk-java-v2/blob/6747fa2abf24d9a0b686f6be3dbe2710719677eb/test/protocol-tests-core/src/main/resources/software/amazon/awssdk/protocol/suites/cases/json-core-output.json#L553)

Reply:
> Can we please update the PR in modification about this change.

Yes, I have done that as well.

Reply:
Thanks